lib/ui/shared/overflow_menu.dart

```diff
@@ -84,15 +87,65 @@ class _OverflowMenuState extends ConsumerState<OverflowMenu> {
   void _enterFullscreen() =>
       ref.read(WorkspaceState.provider.notifier).toggleFullscreen(true);
 
+  Future<bool> _showOverWriteDialog() async {
+    return await showConfirmationDialog(context) ?? false;
+  }
+
+  Future<bool> _deleteFileAndProject(CatrobatImageMetaData imageData,
+      ProjectDatabase db, IFileService fileService) async {
+    final fileName = '${imageData.name}.${imageData.format.extension}';
+
+    final result = await fileService.deleteFileInApplicationDirectory(fileName);
+    if (result is Err) {
+      Fluttertoast.showToast(msg: 'Overwriting failed!');
+      return false;
+    }
+
+    Fluttertoast.showToast(msg: 'Overwriting file...');
```

Comment:
this also does not make sese here?

Reply:
Removed. A toast is already shown if the image is saved successfully in ``ìo_handler.dart``` in ```_saveAsCatrobatImage```.